polkadot/xcm/docs/src/cookbook/other_parachain_tokens/parachain/xcm_config.rs

```diff
@@ -0,0 +1,214 @@
+// Copyright Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! # XCM Configuration
+
+use frame::{
+	deps::frame_system,
+	runtime::prelude::*,
+	traits::{Everything, Nothing},
+};
+use xcm::v4::prelude::*;
+use xcm_builder::{
+	AccountId32Aliases, DescribeAllTerminal, DescribeFamily, EnsureXcmOrigin,
+	FrameTransactionalProcessor, HashedDescription, IsConcrete,
+	SignedToAccountId32,
+};
+use xcm_executor::XcmExecutor;
+
+use super::{AccountId, Balances, MessageQueue, Runtime, RuntimeCall, RuntimeEvent, RuntimeOrigin};
+
+parameter_types! {
+	pub RelayLocation: Location = Location::parent();
+	pub ThisNetwork: NetworkId = NetworkId::Polkadot;
+}
+
+pub type LocationToAccountId = (
+	HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>,
+	AccountId32Aliases<ThisNetwork, AccountId>,
+);
+
+/// Configuration related to asset transactors
+#[docify::export]
+mod asset_transactor {
+	use super::*;
+
+	use frame::traits::EverythingBut;
```

Comment:
To reduce the size of the docify stuff, we can move the imports to `super::*`?

Reply:
I wanted to show what imports are needed, but yeah readers can just go to the code and check it out there if it's not included in the prelude.